imaplugin/src/main/java/com/kaltura/playkit/plugins/ima/IMAPlugin.java

```diff
@@ -937,6 +938,9 @@ private AdInfo createAdInfo(Ad ad) {
         String dealId = ad.getDealId();
         String surveyUrl = ad.getSurveyUrl() != null ? ad.getSurveyUrl() : "";
         String traffickingParams = ad.getTraffickingParameters();
+        List<String> adWrapperCreativeIds = Arrays.asList(ad.getAdWrapperCreativeIds());
```

Comment:
I think better use this pattern for all 3 attributes both IMA and DAI

List<String> adWrapperCreativeIds = ad.getAdWrapperCreativeIds() != null ? Arrays.asList(ad.getAdWrapperCreativeIds()) : Collection.emptyList()